userspace/engine/falco_engine.cpp

```diff
@@ -37,6 +37,7 @@ extern "C" {
 
 string lua_on_event = "on_event";
 string lua_print_stats = "print_stats";
+const std::string falco_engine::m_default_ruleset = "falco-default-ruleset";
```

Comment:
A fairly weak convention we've used for static members of classes is to use a `s_` prefix instead of `m_`. Could you use that instead?

Reply:
Thanks for pointing it out; will fix it asap!

Reply:
Fixed.